client/src/app/app.module.ts

```diff
@@ -17,7 +16,6 @@ import { CookieService } from 'ngx-cookie-service';
     BrowserModule,
     BrowserAnimationsModule,
     AuthModule,
-    AuthRoutingModule,
```

Comment:
Why did you delete the routing of auth? keep it even if there is only one route inside we might add other after related to authentication. And we don't want to mix contexts.

Reply:
I did not think it was necessary because it was only 1 route and to be able to see the whole picture at once. I can add I back if you think we will add more routes there later.

Reply:
Yeah, I would just prefer to keep it in a separate route folder.